modules/http/http-plugin.h

```diff
@@ -46,6 +46,7 @@ typedef struct
   int ssl_version;
   gboolean peer_verify;
   short int method_type;
+  gulong timeout;
   LogTemplate *body_template;
```

Comment:
In the libcurl documentation I saw long:
https://curl.haxx.se/libcurl/c/CURLOPT_TIMEOUT.html
CURLcode curl_easy_setopt(CURL *handle, CURLOPT_TIMEOUT, long timeout),
that's why I used gulong. I will change this to glong from gulong, but I would leave a long-type here to align to the curl documentation.